src/styles/blocks/star.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--rating-4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
```

Comment:
Consider using SCSS variables for repeated values such as image URLs. This makes it easier to change the image path in one place without having to modify each individual usage. For example, you could define a variable like `$star-image: url('../images/star.svg');` at the top of your file and then use it for `background-image` properties.